### PR TITLE
Bug 1447263 - Provide better docs

### PR DESCRIPTION
**Pull request by JohanLorenzo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+```
+
+Ordinarily, scriptworker would get the task definition from TaskCluster, and write it to a `task.json` in the `work_dir`.  Since you're initially not going to run through scriptworker, you need to put this file on disk yourself.
+
+The important entries to edit are the:
+ * `upstreamArtifacts`: point to the file(s) to publish to Google Play
+ * `dependencies`: need to match the `taskId`s of the URLs unless you modify the `valid_artifact_*` config items as specified above
+ * `scopes`: the first and only scope, `project:releng:snapcraft:*`, tells which channel on Snap store should be updated. For more details about scopes. See `scopes.md`
+
+#### (nighlty, beta, release) vs (edge, beta, candidate, release)?
```
Comment (escapewindow) [inline README.md]: nightly

File: scopes.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Scopes expected by pushsnapscript
+
+* `project:releng:snapcraft:firefox:{candidate,beta,mock}`: This represents the Snap store channel pushsnap script publishes the snap to. pushsnapscript only accepts a subset of what the [Snap store allows](https://docs.snapcraft.io/reference/channels#risk-levels-meaning).
```
Comment (escapewindow) [inline scopes.md]: are you able to either link to or move the `(nightly, beta, release) vs (edge, beta, candidate, release)?` section here?

Comment (JohanLorenzo) [reply scopes.md]: Good idea. Moved.

File: README.md (hunk 1 above)
Comment (JohanLorenzo) [reply README.md]: Fixed

**Pull request merged**